javalib/src/main/scala/java/lang/ClassLoader.scala

```diff
@@ -1,15 +1,9 @@
 package java.lang
 
-import scalanative.annotation.stub
-
 class ClassLoader protected (parent: ClassLoader) {
   def this() = this(null)
-  @stub
-  def loadClass(name: String): Class[_] = ???
-  @stub
-  def getParent(): ClassLoader = ???
-  @stub
-  def getResourceAsStream(name: String): java.io.InputStream = ???
-  @stub
-  def getResources(name: String): java.util.Enumeration[_] = ???
+  // def loadClass(name: String): Class[_] = ???
+  // def getParent(): ClassLoader = ???
+  // def getResourceAsStream(name: String): java.io.InputStream = ???
+  // def getResources(name: String): java.util.Enumeration[_] = ???
```

Comment:
Should we remove this class itself?
The `ClassLoader` class with no members might make it hard to stub by users?

Reply:
We do have a `Class.getResourceAsStream` so normally you can do that with a `ClassLoader` too and also it is not clear why the other `getResource` methods are not implemented. Not sure if that would hurt porting efforts. Normally I would say we should remove it but not 100% sure because `FileSystems` has a couple of methods needing a `ClassLoader`.

Reply:
Well, I answered my own question - they are not implemented because the return `URL` or `Enumeration[URL]`.